src/main/java/com/learning/pages/swaglabs/SwagLabsInventory.java

```diff
@@ -0,0 +1,25 @@
+package com.learning.pages.swaglabs;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class SwagLabsInventory {
+
+  public static final String INVENTORY_PAGE_URL = "https://www.saucedemo.com/inventory.html";
+
+  //TODO
+  // find addToCartButton without arrays
+  @FindBy(css = "a#item_1_img_link")
+  private WebElement addToCartButton;
```

Comment:
Please show me in code solution for this :)

Reply:
I'll try doing this today/tomorrow.

Reply:
I'm going to give you a pseudo code here.

In this class you'd have:

```java
@FindBy(css = "<locator of whole offering card element>")
 private List<WebElement> offeringCard;

public List<OfferingCard> getOfferingCards(){
return offeringCard.stream().map(OfferingCard::new).collect(Collectors.toList())
}
```

Then you'd have to create new `OfferingCard` element which should not extend `BasePage` because this class does not represent a page, but just an element on that page.

```java
public class OfferingCard {
    @FindBy(//locator of that button)
    private WebElement addToCartButton;

    //Other object elements

    OfferingCard(WebElement rootElement) {
        PageFactory.initElements(new DefaultElementLocatorFactory(rootElement, this))
    }

    public void addToCart() {
        addToCartButton.click();
    }
}
```

Reply:
I will implement it at Friday.